src/Illuminate/Console/Scheduling/Event.php

```diff
@@ -719,7 +729,7 @@ public function withoutOverlapping()
         $this->withoutOverlapping = true;
 
         return $this->skip(function () {
-            return file_exists($this->mutexPath());
+            return $this->cache->has($this->mutexName());
```

Comment:
@themsaid I believe this is the only line that would need to change to make it fully BC. Something like:

```php
return $this->cache->has($this->mutexName()) || file_exists($this->mutexPath());
```